### PR TITLE
change fastcgi_cache_key default false->undef

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
merge this PR before we merge https://github.com/voxpupuli/puppet-nginx/pull/1031

**Comment by bastelfreak** [comment]
this fixes https://github.com/voxpupuli/puppet-nginx/issues/1048

File: manifests/init.pp [hunk 1]
```diff
@@ -217,9 +217,6 @@
   validate_string($fastcgi_cache_keys_zone)
   validate_string($fastcgi_cache_max_size)
   validate_string($fastcgi_cache_inactive)
-  if ($fastcgi_cache_key != false) {
```
Comment (alexjfisher) [inline manifests/init.pp]: @bastelfreak You won't forget to add the validation back in in another PR?  Or maybe you should just put the new validation in using a data type in this PR too.  Users don't really care how the validation is performed.

Comment (bastelfreak) [inline manifests/init.pp]: that is already handled in https://github.com/voxpupuli/puppet-nginx/pull/1031

**Pull request closed without merging**